data/formations.txt

```diff
@@ -0,0 +1,300 @@
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+# Atomic formation that doesn't rotate with the lead ship.
+formation "Atomic"
+	rotatable 120
+	# Section in front of the lead ship.
+	arc
+		start -75 -129.9
+		angle 60
+		positions 2
+		repeat
+			start -75 -129.9
+			positions 1
+	# Side sections.
+	arc
+		start 150 0
+		angle 60
+		positions 2
+		repeat
+			start 150 0
+			positions 1
+	arc
+		start -150 0
+		angle -60
+		positions 2
+		repeat
+			start -150 0
+			positions 1
+
+
+# A filled circle around the lead ship.
+# Built in six segment sections using the same layouting as the hexagon shape.
+# Using rotatable 1 instead of 0 to avoid aligning circles perfectly with the screen direction.
+formation "Circle"
+	rotatable 1
+	arc
+		start 200 0
+		angle 360
+		# Start and end positions overlap, so we need to specify 7 instead of 6 and use skip first or skip last.
+		# No anchor point specified, since the flagship (position 0,0) is the anchor point (and that is the default).
+		skip last
+		positions 7
+		repeat
+			start 200 0
+			positions 6
+
+
+# Triangle forming behind the formation leader.
+formation "Delta (tailing)"
+	flippable y
+	line
+		start -75 129.9
+		end 75 129.9
+		positions 2
+		repeat
+			start -75 129.9
+			end 75 129.9
+			positions 1
+
+
+formation "Diamond"
+	flippable x y
+	rotatable 90
+	# Segment below, right of lead ship. Growing in down/right direction.
+	line
+		start 100 100
+		end 0 200
+		positions 2
+		repeat
+			start 200 0
+			end 0 200
+			positions 2
+	# Segment left, below of lead ship. Growing in left/down direction.
+	line
+		start -100 100
+		end -200 0
+		positions 2
+		repeat
+			start 0 200
+			end -200 0
+			positions 2
+	# Segment above, left of lead ship. Growing in above/left direction.
+	line
+		start -100 -100
+		end 0 -200
+		positions 2
+		repeat
+			start -200 0
+			end 0 -200
+			positions 2
+	# Segment right, above of lead ship. Growing in right/above direction.
+	line
+		start 100 -100
+		end 200 0
+		positions 2
+		repeat
+			start 0 -200
+			end 200 0
+			positions 2
+
+
+# Formation where ships fly in a diagonal line left behind the formation leader.
+formation "Echelon (left)"
+	position -150 150
+		repeat -150 150
+
+
+# Formation where ships fly in a diagonal line right behind the formation leader.
+formation "Echelon (right)"
+	position 150 150
+		repeat 150 150
+
+
+formation "Hexagon (wide)"
```

Comment:
```suggestion
formation "Hexagon"
```

Reply:
I originally wanted to have 2 hexagon formations:
- A hexagon with points to the sides and flat sides front and after (relative to flagship)
- A hexagon with points to the front and back and flat sides on the sides (relative to flagship)

This hexagon is the first one, and thus post-fixed with "wide".
The second one would be post-fixed with "long".

Do you have suggestions for better names?

Reply:
It might just be easier to have a single stable formation. Having two formations that only differ in 60 degrees of rotation seems overkill.

Reply:
You're right, having two nearly identical formations is probably confusing.